src/routes/_error.svelte

```diff
@@ -1,15 +1,67 @@
 <script>
+	import Button from '../components/common/Button.svelte';
+	import DashBackground from '../components/DashBackground.svelte';
+	import Section from '../components/Section.svelte';
+
 	export let error;
 	export let status;
+
+	const stack = error.stack;
 </script>
 
+<style>
+	.h1 {
+		font-size: 14rem;
+		word-break: keep-all;
+		-webkit-text-stroke: 2px black;
+	}
+
+	.card {
+		border-radius: 15px;
```

Comment:
Tailwinds "rounded-2xl" doesn't work here. I don't know why

Reply:
I'll be looking into this after merging it into master.